### PR TITLE
chore(docs): initialize contributor docs for content releases

### PR DESCRIPTION
**Pull request by markkaylor**
### What does it do?

Initialize the contributor documentation and documents everything for the backend currently on the feature branch

### Why is it needed?

So we can continue documenting the feature as we develop

### How to test it?

Just give it a read

### CODE REVIEW

File: docs/docs/docs/01-core/content-releases/00-intro.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+```
+packages/core/content-releases
+```
+
+```mdx-code-block
+import DocCardList from '@theme/DocCardList';
+import { useCurrentSidebarCategory } from '@docusaurus/theme-common';
+
+<DocCardList items={useCurrentSidebarCategory().items} />
+```
```
Comment (remidej) [inline docs/docs/docs/01-core/content-releases/00-intro.md:24]: you dropped this

File: docs/docs/docs/01-core/content-releases/01-backend.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+This content type stores all the information about a release and its associated Release Actions. It is saved in the database as `strapi_release`. The schema can be found in:
+
+```
+packages/core/content-releases/server/src/content-types/release/schema.ts
+```
+
+### Release Action
+
+This content type is associated with any entry from any content-type with draft and publish enable. It is also responsible for storing the action to perform for the associated entry. It is saved in the database as `strapi_release_actions`. The schema can be found in:
```
Comment (remidej) [inline docs/docs/docs/01-core/content-releases/01-backend.md:29]: this one has a plural db name, the other a singular one, I assume 1 of the 2 is wrong

**Review by remidej — COMMENTED** [review]
good idea to start this early 👍

File: docs/docs/docs/01-core/content-releases/01-backend.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+**Get all releases**:
+
+- method: `GET`
+- endpoint: `/content-releases/`
+
+**Create a release**:
+
+- method: `POST`
+- endpoint: `/content-releases/`
```
Comment (Feranchz) [inline docs/docs/docs/01-core/content-releases/01-backend.md:55]: Should we specify the body of this request? Even if it's really simple I think could be useful:
```
body: { name }
```

File: docs/docs/docs/01-core/content-releases/01-backend.md (hunk 2 above)
Comment (markkaylor) [reply docs/docs/docs/01-core/content-releases/01-backend.md:29]: yeah they should both be plural 👍

File: docs/docs/docs/01-core/content-releases/00-intro.md (hunk 1 above)
Comment (markkaylor) [reply docs/docs/docs/01-core/content-releases/00-intro.md:24]: Sorry not sure I understand

File: docs/docs/docs/01-core/content-releases/01-backend.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+Release routes can be found in:
+
+```
+packages/core/content-releases/server/src/routes/release.ts
+```
+
+**Get all releases**:
+
+- method: `GET`
+- endpoint: `/content-releases/`
```
Comment (Feranchz) [inline docs/docs/docs/01-core/content-releases/01-backend.md:50]: We can mention here that this request works with pagination? So you can pass page and pageSize as query params?

**Review by remidej — APPROVED** [review]

**Pull request merged**